addons/website/static/lib/multirange/multirange_custom.js

```diff
@@ -295,11 +295,11 @@ class Multirange {
     }
 
     formatNumber(number) {
-        number = String(number).split('.');
-        if (number[1] && number[1].length === 1) {
-            number[1] += '0';
-        }
-        let formatedNumber = number[0].replace(/(?=(?:\d{3})+$)(?!\b)/g, ',') + (number[1] ? '.' + number[1] : '.00');
+        const locale = document.querySelector("html").getAttribute("lang");
+        let formatedNumber = number.toLocaleString(locale, {
```

Comment:
While you are at it, you may want to fix the typo: format**T**edNumber.
I think you could also make the variable a `const`:
```js
const formattedNumber = number.toLocaleString(locale, {
    minimumFractionDigits: 2,
    maximumFractionDigits: 2,
});
if (!this.currency) {
    return formattedNumber;
}
if (this.currenyPosition === "after") {
    return `${formattedNumber} ${this.currency}`;
}
return `${this.currency} ${formattedNumber}`;
```